src/crimson/osd/osd_operations/background_recovery.cc

```diff
@@ -180,10 +186,11 @@ PGPeeringPipeline &BackfillRecovery::peering_pp(PG &pg)
 BackfillRecovery::interruptible_future<bool>
 BackfillRecovery::do_recovery()
 {
-  logger().debug("{}", __func__);
+  LOG_PREFIX(BackfillRecovery::do_recovery);
+  DEBUGI("{}", __func__);
 
   if (pg->has_reset_since(epoch_started)) {
-    logger().debug("{}: pg got reset since epoch_started={}",
+    DEBUGI("{}: pg got reset since epoch_started={}",
```

Comment:
I think all of the logs in this file should be DPP log lines so that the PG prefix part is both mandatory and uniform.  Can we swap the I variants to DPPI?